io/disk/lvsetup.py

```diff
@@ -45,11 +46,19 @@ def setUp(self):
         """
         Check existence of input PV,VG, LV and snapshots prior to Test.
         """
+        pkg = " "
+        smm = SoftwareManager()
         self.disk = self.params.get('disk', default=None)
         vg_name = self.params.get('vg_name', default='avocado_vg')
         lv_name = self.params.get('lv_name', default='avocado_lv')
         self.lv_size = self.params.get('lv_size', default='1G')
-        self.fs_name = self.params.get('fs', default='ext4')
+        self.fs_name = self.params.get('fs', default='ext4').lower()
+        if self.fs_name.lower() == 'xfs':
```

Comment:
self.fs_name should be sufficient as you already did  lower conversion 

`self.fs_name = self.params.get('fs', default='ext4').lower()`